.tekton/exhort-pull-request.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-event: '[pull_request]'
     pipelinesascode.tekton.dev/on-target-branch: '[stable]'
+    pipelinesascode.tekton.dev/on-cel-expression: |
```

Comment:
when using `on-cel-expression` the `on-target-branch` is ignored so you should remove it